__tests__/app.test.js

```diff
@@ -316,6 +318,82 @@ describe("app", () => {
             });
         });
       });
+      describe("PATCH /articles/:article_id", () => {
+        test("200: returns the article with updated votes value", () => {
+          return request(app)
+            .patch("/api/articles/1")
+            .send({inc_votes: 1})
+            .expect(200)
+            .then(({ body }) => {
+              expect(body.article).toEqual({
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 101,
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              })
+            });
+        });
+        test("200: returns the article with updated votes value when votes are negative", () => {
+          return request(app)
+            .patch("/api/articles/1")
+            .send({inc_votes: -10})
+            .expect(200)
+            .then(({ body }) => {
+              expect(body.article).toEqual({
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 90,
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              })
+            });
+        });
+        test("404: article valid id but does not exist", () => {
+          return request(app)
+            .patch("/api/articles/1001")
+            .send({inc_votes: -10})
+            .expect(404)
+            .then(({ body }) => {
+              expect(body.msg).toEqual('Article Not Found')
+            });
+        });
+        test("400: article invalid id", () => {
+          return request(app)
+            .patch("/api/articles/DROP TABLE users")
+            .send({inc_votes: -10})
+            .expect(400)
+            .then(({ body }) => {
+              expect(body.msg).toEqual('Bad Request')
+            });
+        });
+        test("400: no inc_votes passed", () => {
+          return request(app)
+            .patch("/api/articles/DROP TABLE users")
+            .send({updatedVotes: -10})
+            .expect(400)
+            .then(({ body }) => {
+              expect(body.msg).toEqual('Bad Request')
+            });
```

Comment:
I think it would still send back the comment, but it would be unchanged - potentially? This would be a 200